tag_release

```diff
@@ -82,9 +82,12 @@ if [[ -L release.xml ]]; then
 fi
 
 # Generate a release manifest with all revisions pinned
-repo manifest -o release.xml -r
+tmp=$(tempfile --prefix repo)
```

Comment:
From `tempfile(1)`: `tempfile creates a temporary file in a safe manner.  It uses tempnam(3)` and later `tempfile is deprecated; you should use mktemp(1) instead.`

And from `tempnam(3)`: `Never use this function.  Use mkstemp(3) or tmpfile(3) instead.`

Yay legacy!